test/helpers/rebalancingSetBidderHelpers.ts

```diff
@@ -0,0 +1,54 @@
+import * as _ from 'lodash';
+import Web3 from 'web3';
+import { Address } from 'set-protocol-utils';
+import { RebalancingSetEthBidder } from 'set-protocol-contracts';
+import {
+  ERC20Wrapper,
+  RebalancingSetEthBidderContract,
+  RebalanceAuctionModuleContract,
+  TransferProxyContract,
+  WethMockContract,
+} from 'set-protocol-contracts';
+
+import {
+  DEFAULT_ACCOUNT,
+  TX_DEFAULTS,
+} from '@src/constants';
+
+const contract = require('truffle-contract');
+
+export const deployRebalancingSetEthBidderAsync = async (
+  web3: Web3,
+  rebalanceAuctionModule: RebalanceAuctionModuleContract,
+  transferProxy: TransferProxyContract,
+  wrappedEther: WethMockContract,
+  owner: Address = DEFAULT_ACCOUNT,
+): Promise<RebalancingSetEthBidderContract> => {
+  const truffleRebalancingSetEthBidderContract = contract(RebalancingSetEthBidder);
```

Comment:
We might have a helper that abstracts all these steps away.

`setDefaultTruffleContract` in coreHelpers. If you have time, would be good to refactor all the Helpers to use this

Reply:
Done